textual_fspicker/file_dialog.py

```diff
@@ -160,8 +161,10 @@ def _confirm_file(self, event: Input.Submitted | Button.Pressed) -> None:
         # doing a "cd".
         try:
             if chosen.is_dir():
+                drive = MakePath.of(file_name.value).drive
                 self.query_one(Input).value = ""
                 self.query_one(DirectoryNavigation).location = chosen
+                self.query_one(DriveNavigation).drive = drive
```

Comment:
This code should query if `DriveNavigation` exists at all, and the code to get the drive a couple of lines above should probably depend on that being the code too. Currently this causes a crash on anything other than Windows.

Reply:
Amateur hour 😫. I'll fix this.

Reply:
Fixed in fe41a0e!